Testing file cleanup sporadically fail on under GitHub actions

https://github.com/terjekv/hubuum/actions/runs/5063276650/jobs/9089708381#step:6:27 sporadically gives:

````
python38-django32-sqlite: commands[0]> pytest -n auto /home/runner/work/hubuum/hubuum/hubuum/tests/ /home/runner/work/hubuum/hubuum/hubuum/api/v1/tests/
  ============================= test session starts ==============================
  platform linux -- Python 3.8.16, pytest-7.3.1, pluggy-1.0.0
  cachedir: .tox/python38-django32-sqlite/.pytest_cache
  django: settings: hubuumsite.settings (from env)
  rootdir: /home/runner/work/hubuum/hubuum
  plugins: django-4.5.2, xdist-3.3.1
  created: 2/2 workers
  2 workers [141 items]
  
  ........................................................................ [ 51%]
  ...................................................F.................    [100%]
  =================================== FAILURES ===================================
  ____________ TesttearDownModuleTestCase.test_teardown_with_folders _____________
  [gw1] linux -- Python 3.8.16 /home/runner/work/hubuum/hubuum/.tox/python38-django32-sqlite/bin/python
  
  self = <hubuum.api.v1.tests.test_31_attachments.TesttearDownModuleTestCase testMethod=test_teardown_with_folders>
  
      def test_teardown_with_folders(self):
          """Test teardown with folders."""
  >       with open("attachments/file/test.txt", "w", encoding="utf-8") as file:
  E       FileNotFoundError: [Errno 2] No such file or directory: 'attachments/file/test.txt'
  
  hubuum/api/v1/tests/test_31_attachments.py:519: FileNotFoundError
  =========================== short test summary info ============================
  FAILED hubuum/api/v1/tests/test_31_attachments.py::TesttearDownModuleTestCase::test_teardown_with_folders
  ======================== 1 failed, 140 passed in 8.73s =========================
  python38-django32-sqlite: exit 1 (10.45 seconds) /home/runner/work/hubuum/hubuum> pytest -n auto /home/runner/work/hubuum/hubuum/hubuum/tests/ /home/runner/work/hubuum/hubuum/hubuum/api/v1/tests/ pid=2623
python38-django32-sqlite: FAIL ✖ in 45.91 seconds
````

--- a/hubuum/api/v1/tests/test_31_attachments.py
+++ b/hubuum/api/v1/tests/test_31_attachments.py
@@ -511,36 +511,7 @@ class HubuumAttachmentBasicTestCase(HubuumAttachmentTestCase):
                     )
 
 
-class TesttearDownModuleTestCase(HubuumAPITestCase):
-    """Test case for global teardown of this module."""
-
-    def test_teardown_with_folders(self):
-        """Test teardown with folders."""
-        with open("attachments/file/test.txt", "w", encoding="utf-8") as file:
-            file.write("Hello, world!")
-        _cleanup_attachment_directories()
-
-    def test_teardown_without_folders(self):
-        """Test teardown without folders."""
-        # Run twice to ensure we have no folders to delete.
-        _cleanup_attachment_directories()
-        _cleanup_attachment_directories()
-
-
-def _cleanup_attachment_directories() -> None:
-    """Cleanup the attachments directories."""
-    _cleanup_directory("attachments")
-    _cleanup_directory("attachments/file")
-
-
-def _cleanup_directory(directory: str) -> None:
-    """Cleanup the attachments directory."""
-    try:
-        os.rmdir(directory)
-    except OSError:
-        pass
-
-
-def tearDownModule():  # pylint: disable=invalid-name
+def tearDownModule():  # pylint: disable=invalid-name # pragma: no cover
     """Global teardown for this test module, cleans up attachments directory."""
-    _cleanup_attachment_directories()
+    for file in os.scandir("attachments/file/"):
+        os.remove(file.path)

